packages/@sanity/portable-text-editor/src/utils/__tests__/values.test.ts

```diff
@@ -72,7 +72,6 @@ Array [
         "text": "123",
       },
     ],
-    "markDefs": Array [],
```

Comment:
I may be lost here, but shouldn't this be always present when we have a marks in a children? Why this was removed here?

Reply:
The schema actually conciders this property optional. And additionally we need to be able to test if the value has changed regarding this prop (for example when a validation resolution is performed that inserts empty markDefs) we need to be able to make this distinction.